pkg/store/cqlstore.go

```diff
@@ -90,11 +92,25 @@ func (cs *cqlStore) doMutate(ctx context.Context, builder qb.Builder, ts time.Ti
 	return nil
 }
 
-func (cs *cqlStore) load(ctx context.Context, builder qb.Builder, values []interface{}) (result []map[string]interface{}, err error) {
+func (cs *cqlStore) loadSV(ctx context.Context, builder qb.Builder, values []interface{}) (sv.Result, error) {
```

Comment:
SV and MV are cryptic.
Why not expand them here?

Reply:
Expanded